src/main/resources/assets/corpse/lang/en_us.json

```diff
@@ -1,21 +1,21 @@
 {
   "entity.corpse.corpse": "Corpse",
   "entity.corpse.corpse_of": "Corpse of %s",
-  "button.corpse.previous": "Previous",
+  "button.corpse.previous": "Back",
```

Comment:
Back means going back one instance, but it is actually the previous page.

Reply:
If I remember correctly, various installers use "< Back" as a button label, and when clicked, just the previous page is displayed. I think it is a good replacement.